chord_gen_rest.py

```diff
@@ -5,7 +5,7 @@
 
 @app.route('/format_chordpro/', methods=['GET'])
 def home():
-    return "<h1>Good Morning, weirdo!</h1><p>You were fooled by yourself! =D =,C                  inferno</p>"
+    return "<h1>palavra</h1><p>palavras</p>"
```

Comment:
Cool!
But it's still not printing the request which came from the client. 
You must send the text you typed in the input text area and receive it here, print it, then modify it and return it in the response